crates/matrix-sdk/src/client/mod.rs

```diff
@@ -4355,5 +4355,15 @@ pub(crate) mod tests {
             room.matrix_permalink(true).await.unwrap().to_string(),
             "matrix:r/canonical:localhost?action=join"
         );
+
+        let event_id = event_id!("$15139375512JaHAW");
+        assert_eq!(
```

Comment:
A bit annoying to make this test even bigger, though setting up the relevant state is probably even more annoying.